Marlin/kaynak/lcd/dil/language_tr.h

```diff
@@ -256,34 +345,58 @@ namespace Language_tr {
+  LSTR MSG_VC_JERK                        = _UxGT("V") STR_C _UxGT(" Sarsma");
+  LSTR MSG_VN_JERK                        = _UxGT("Max @ Sarsma");
+  LSTR MSG_VE_JERK                        = _UxGT("Max E Sarsma");
   LSTR MSG_JUNCTION_DEVIATION             = _UxGT("Jonksiyon Sapması");
   LSTR MSG_MAX_SPEED                      = _UxGT("Hız Vektörü");
-  LSTR MSG_VMAX_A                         = _UxGT("HızVektör.max ") STR_A;
-  LSTR MSG_VMAX_B                         = _UxGT("HızVektör.max ") STR_B;
-  LSTR MSG_VMAX_C                         = _UxGT("HızVektör.max ") STR_C;
-  LSTR MSG_VMAX_N                         = _UxGT("HızVektör.max @");
-  LSTR MSG_VMAX_E                         = _UxGT("HızVektör.max E");
-  LSTR MSG_VMAX_EN                        = _UxGT("HızVektör.max *");
-  LSTR MSG_VMIN                           = _UxGT("HızVektör.min");
-  LSTR MSG_VTRAV_MIN                      = _UxGT("HV.gezinme min");
+  LSTR MSG_VMAX_A                         = _UxGT("Max ") STR_A _UxGT(" Hız");
+  LSTR MSG_VMAX_B                         = _UxGT("Max ") STR_B _UxGT(" Hız");
+  LSTR MSG_VMAX_C                         = _UxGT("Max ") STR_C _UxGT(" Hız");
+  LSTR MSG_VMAX_N                         = _UxGT("Max @ Hız");
+  LSTR MSG_VMAX_E                         = _UxGT("Max E Hız");
+  LSTR MSG_VMAX_EN                        = _UxGT("Max * Hız");
+  LSTR MSG_VMIN                           = _UxGT("Min Hız");
+  LSTR MSG_VTRAV_MIN                      = _UxGT("Min Gezme Hızı");
   LSTR MSG_ACCELERATION                   = _UxGT("Ivme");
-  LSTR MSG_AMAX_A                         = _UxGT("Max. ivme ") STR_A;
-  LSTR MSG_AMAX_B                         = _UxGT("Max. ivme ") STR_B;
-  LSTR MSG_AMAX_C                         = _UxGT("Max. ivme ") STR_C;
-  LSTR MSG_AMAX_N                         = _UxGT("Max. ivme @");
-  LSTR MSG_AMAX_E                         = _UxGT("Max. ivme E");
-  LSTR MSG_AMAX_EN                        = _UxGT("Max. ivme *");
-  LSTR MSG_A_RETRACT                      = _UxGT("Ivme-geri çekme");
-  LSTR MSG_A_TRAVEL                       = _UxGT("Ivme-gezinme");
+  LSTR MSG_AMAX_A                         = _UxGT("Max ") STR_A _UxGT(" İvme");
+  LSTR MSG_AMAX_B                         = _UxGT("Max ") STR_B _UxGT(" İvme");
+  LSTR MSG_AMAX_C                         = _UxGT("Max ") STR_C _UxGT(" İvme");
+  LSTR MSG_AMAX_N                         = _UxGT("Max @ İvme");
+  LSTR MSG_AMAX_E                         = _UxGT("Max E İvme");
+  LSTR MSG_AMAX_EN                        = _UxGT("Max * İvme");
+  LSTR MSG_A_RETRACT                      = _UxGT("G.Çekme Hızı");
+  LSTR MSG_A_TRAVEL                       = _UxGT("Gezinme Hızı");
+  LSTR MSG_INPUT_SHAPING                  = _UxGT("Girdi Şekillendirme");
+  LSTR MSG_SHAPING_ENABLE                 = _UxGT("@ şekillendirmesini etkinleştir");
+  LSTR MSG_SHAPING_DISABLE                = _UxGT("@ şekillendirmesini devre dışı bırak");
```

Comment:
Meanwhile, these strings are way too long. On character LCD screens users will only see this for both items:
```
@ şekillendirmesin
```